### PR TITLE
match email/n_user for asana/trello/todoist + fixes

### PR DESCRIPTION
**Pull request by mjamroz**

### CODE REVIEW

**Comment by mjamroz** [comment]
tested [V]

File: src/ntimporters/todoist/importer.py [hunk 1]
```diff
@@ -74,7 +75,7 @@ def _import_project(project: dict):
                 author_id=models.Id16ReadOnly(id16()),
                 created_at=models.TimestampReadOnly(1),
                 last_event_at=models.TimestampReadOnly(1),
-                is_favorite=project.favorite,
+                is_favorite=project.is_favorite,
```
Comment (mjamroz) [inline src/ntimporters/todoist/importer.py:78]: i dont understand how it was working before

**Comment by mjamroz** [comment]
@jsamelak  could you take a look in your spare time? Im commenting only to know that you know about this PR (is not to speed up ;) )

File: src/ntimporters/utils.py [hunk 2]
```diff
@@ -238,3 +236,31 @@ def post_tag(nt_client, tag_name: str, color: str):
...
+    """Match 3rd party with Nozbe users and return email,member id pairs"""
+
+    def md5(email: str, user_id: str):
+        "Calculate hash"
+        return hashlib.md5((user_id + email.lower()).encode(encoding="utf-8")).hexdigest()  # nosec
+
+    nt_users = [
+        (str(elt.email), str(elt.id))
+        for elt in apis.UsersApi(nt_client).get_users()
+        if hasattr(elt, "email")
```
Comment (jsamelak) [inline src/ntimporters/utils.py]: placeholder users may have email stored in `invitation_email` field, would be good to match them as well

**Review by jsamelak — APPROVED** [review]

File: src/ntimporters/todoist/importer.py (hunk 1 above)
Comment (mjamroz) [reply src/ntimporters/todoist/importer.py:78]: https://developer.todoist.com/rest/v2/?python#migrating-from-v1 i know - they changed name.....

**Pull request merged**